poc/policy/histogram_with_aggregator_dp.py

```diff
@@ -0,0 +1,82 @@
+"""
+A policy that uses only Aggregator-DP to achieve central (eps, delta)-DP.
```

Comment:
For consistency with the draft. (Or, if you like update the draft to use `eps` and `delta`. Go with whatever feels right!) Also, I like to use backticks around variables I refer to in comments. Just a suggestion, again totally up to you.

Here and below.

```suggestion
A policy that uses only Aggregator-DP to achieve central (`EPSILON`, `DELTA`)-DP.
```

Reply:
Sounds right. Let's use `EPSILON` and `DELTA` in docstring. We can even update it if we end up using Greek letters.